Packs/CommonPlaybooks/Playbooks/playbook-Block_Email_-_Generic_v2.yml

```diff
@@ -0,0 +1,382 @@
+id: Block Email - Generic v2
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
+name: Block Email - Generic v2
+description: This playbook will block emails at your mail relay integration.
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: e4cf2a65-fab2-48c9-8d52-479632660700
+    type: start
+    task:
+      id: e4cf2a65-fab2-48c9-8d52-479632660700
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      "#none#":
+      - "3"
+      - "4"
+      - "5"
+      - "6"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 750,
+          "y": -30
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 335f96a3-ea62-4a3f-8808-873c57caa695
+    type: title
+    task:
+      id: 335f96a3-ea62-4a3f-8808-873c57caa695
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 740,
+          "y": 500
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: b30d383c-cdfb-4200-8ed6-cb7ab5b00a1c
+    type: playbook
+    task:
+      id: b30d383c-cdfb-4200-8ed6-cb7ab5b00a1c
+      version: -1
+      name: Symantec block Email
+      playbookName: Symantec block Email
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      "#none#":
+      - "2"
+    scriptarguments:
+      EmailToBlock:
+        simple: ${inputs.EmailToBlock}
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 130,
+          "y": 120
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
```

Comment:
Did not fully review, but just at a quick glance, please make sure all of the `skipunavailable` are true, otherwise it'll break the generic playbooks if any pack is missing for anyone